mach_eval/analyzers/electromagnetic/winding_factors.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+class WindingFactorsProblem:
+    """Problem class for winding factor analyzer
+    Attributes:
+        harmonics: number of harmonics to be included in calculations []
```

Comment:
`harmonics:` I think you mean for this to be a *list* of harmonics to calculate winding factors at